kotlin/samples/are-you-sure/src/main/java/com/squareup/sample/areyousure/AreYouSureWorkflow.kt

```diff
@@ -0,0 +1,78 @@
+package com.squareup.sample.areyousure
+
+import com.squareup.sample.areyousure.AreYouSureWorkflow.Finished
+import com.squareup.sample.areyousure.AreYouSureWorkflow.State
+import com.squareup.sample.areyousure.AreYouSureWorkflow.State.Quitting
+import com.squareup.sample.areyousure.AreYouSureWorkflow.State.Running
+import com.squareup.sample.container.BackButtonScreen
+import com.squareup.workflow.RenderContext
+import com.squareup.workflow.Snapshot
+import com.squareup.workflow.StatefulWorkflow
+import com.squareup.workflow.WorkflowAction.Companion.noAction
+import com.squareup.workflow.action
+import com.squareup.workflow.ui.AlertContainerScreen
+import com.squareup.workflow.ui.AlertScreen
+import com.squareup.workflow.ui.AlertScreen.Button.NEGATIVE
+import com.squareup.workflow.ui.AlertScreen.Button.POSITIVE
+import com.squareup.workflow.ui.AlertScreen.Event.ButtonClicked
+import com.squareup.workflow.ui.AlertScreen.Event.Canceled
+
+typealias AreYouSureScreen = AlertContainerScreen<*>
```

Comment:
`typealias` is probably silly. Originally it was `AlertContainerScreen<BackButtonScreen<*>>`, but that was cumbersome to work with.

Reply:
Yea, I think this just makes it harder to read.